fms/testing/_internal/model_test_suite.py

```diff
@@ -64,10 +64,11 @@ def model(self, uninitialized_model: nn.Module):
         params = sorted(sd.keys())
         for key in params:
             parameter = sd[key]
-            values = torch.randn_like(parameter)
-            values -= 0.5
-            values /= 20.0
-            parameter.copy_(values)
+            if not isinstance(parameter, torch.LongTensor):
```

Comment:
which parameters are LongTensor?

Reply:
This will be removed, it used to be the pos_id but we have since removed them